<?xml version="1.0" encoding="UTF-8"?> 
<launch>
    
    <arg name="can_interface_name"/>
    <arg name="published_ros_topic_name" default=""/>
    <arg name="subscribed_ros_topic_name" default=""/>

    <!-- Launch tf publisher node -->
    <node name="tcan_bridge_$(arg can_interface_name)" pkg="tcan_bridge_ros1" type="bidirectional_bridge" output="screen">
        <param name="can_interface_name" value="$(arg can_interface_name)"/>
        <param name="published_ros_topic_name" value="$(arg published_ros_topic_name)"/>
        <param name="subscribed_ros_topic_name" value="$(arg subscribed_ros_topic_name)"/>
    </node>
</launch>
